### PR TITLE
Added an event that is emitted when a channel is created.

### PR DESCRIPTION
**Pull request by Gongreg**
Issue: Need ability to know when a channel was created between server and manager.

## What I did
Added an event (copied logic from #1639).

### CODE REVIEW

File: app/react-native/src/manager/provider.js [hunk 1]
```diff
@@ -42,16 +47,6 @@ export default class ReactProvider extends Provider {
     const renderPreview = addons.getPreview();
 
     const innerPreview = renderPreview ? renderPreview(kind, story) : null;
-
-    if (this.options.manualId) {
```
Comment (shilman) [inline app/react-native/src/manager/provider.js]: @Gongreg Was this just for debugging? Any implications to removing it?

**Review by shilman — APPROVED** [review]
This all looks pretty reasonable to me. @tmeasday any thoughts?

File: app/react-native/src/manager/provider.js (hunk 1 above)
Comment (Gongreg) [reply app/react-native/src/manager/provider.js]: @shilman Kinda. That is what I used before to connect phone with browser.

**Comment by tmeasday** [comment]
Seems reasonable to me. Can we use this to fix https://github.com/storybooks/storybook/issues/1192?

**Review by ndelangen — APPROVED** [review]

**Comment by Gongreg** [comment]
@shilman we should try it out and create a separate pr i guess. :)

**Pull request merged**